2000-10-23  Jakub Jelinek  <jakub@redhat.com>

	* g++.old-deja/g++.other/inline16.C: New test.

--- gcc/testsuite/g++.old-deja/g++.other/inline16.C.jj	Mon Oct 23 16:06:28 2000
+++ gcc/testsuite/g++.old-deja/g++.other/inline16.C	Mon Oct 23 16:06:58 2000
@@ -0,0 +1,24 @@
+// Build don't link:
+// Origin: Jakub Jelinek <jakub@redhat.com>
+// Special g++ Options: -O1
+
+struct foo {
+  bool x;
+  inline void a (unsigned char y);
+  inline void b (void);
+  virtual ~foo ();
+};
+
+foo::~foo ()
+{
+}
+
+void foo::a (unsigned char y)
+{
+    x = ((y & 2) != 0);
+}
+
+void foo::b (void)
+{
+    a(0x07);
+}
